### PR TITLE
Feature/185 convert find view

### LINKED ISSUE #185: Convert Find view to vuetify.
**Is your feature request related to a problem? Please describe.**
We need to make the view readable. It is still using the old bootstrap standard.

**DoD**
- The user can switch between searching for loans or users. 
- The Find view has been converted to vuetify.
- Uses SearchList component.

**Additional context**
Relates to #133 
Relates to #143

### PR DESCRIPTION
**Pull request by TE4-Daniel-Kull**
## What issue did you implement or fix?
Fixes #185

## Summary
- Refactored FindView.
- Implemented actions for SearchListComponent.
- Removed AllBooks, MainTable, DropdownFind. (These have been replaced by FindView and SearchListComponent) 

## Type of change
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes

## How can this implementation be improved?
- Use a slot and template solution inside SearchListComponent instead of my hardcoded actions.
- Implement backend functionality for fetching loans, books and users into FindView.
- Use a different approach for setting the route of each book.

### CODE REVIEW

File: frontend/src/views/FindView.vue [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+
+  // get items() {
+  //     const items = Object.entries(LoansModule.all)
+  //         .filter(([key, value]) => {
+  //             return !value.returned_at;
+  //         })
+  //         .map(([key, value]) => Object.assign(value, { '.key': key }));
+
+  //     return items;
+  // }
```
Comment (te4-hugo-svanberg) [inline frontend/src/views/FindView.vue]: Couldn't you remove this and make it a new issue so that we don't have todos inside the code?

Comment (te4-gabriel-stark) [inline frontend/src/views/FindView.vue]: This could be taken out of the code and be made into an issue.

**Comment by itggot-linus-styren** [comment]
This PR removes the DS_Store file since it was present in develop

File: frontend/src/views/FindView.vue [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+
+import LoansModule from '../store/modules/LoansModule';
+import SearchListComponent from '@/components/SearchListComponent.vue';
+
+@Component({
+  components: {
+    SearchListComponent
+  },
+})
+export default class Test extends Vue {
```
Comment (itggot-linus-styren) [inline frontend/src/views/FindView.vue]: This class should be called `FindView`

**Pull request merged**